tests/integration/cocoon/data/port_group_tests/test_10_pg_add_no_new_portfolio.csv

```diff
@@ -0,0 +1,2 @@
+PortGroupCode,PortGroupDisplayName,FundCode,
+PG_TEST10,Test Portfolio Group,TEST_COM1,
```

Comment:
Some of these test files have a leading index column, and others are mssing. Is this necessary/deliberate?